src/Tasks/Microsoft.NET.Build.Tasks/RuntimeOptions.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.NET.Build.Tasks
 {
     internal class RuntimeOptions
     {
+        public string tfm { get; set; }
```

Comment:
Why can't this value be inferred from the existing "Framework" property?  Also, TFMs don't have patch numbers in them.  So if we had a bug fix in crossgen between 1.1.0 and 1.1.1, how would the cache get updated?  The assembly was crossgened using 1.1.0 and cached in "netcoreapp1.1", but when I installed 1.1.1, there is no way to get the cache updated, because the assembly was already cached for "netcoreapp1.1".